app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class UsersController < Api::ApplicationController
```

Comment:
It is better to create base controller in the scope of each API version. Because there can be significant changes between API versions that apply to base controllers. For example, the way how request is authenticated.